metricsaggregator/src/test/java/io/dockstore/metricsaggregator/helper/CpuAggregatorTest.java

```diff
@@ -70,4 +71,10 @@ void testGetAggregatedMetricFromWorkflowExecutions() {
         assertEquals(4.0, cpuMetric.get().getAverage());
         assertEquals(3, cpuMetric.get().getNumberOfDataPointsForAverage());
     }
+    
+    private RunExecution createRunExecution(Integer cpuRequirements) {
```

Comment:
Kinda liked the old way, could use a proper builder object design pattern though.
Could be followup ticket.